src/cmdc/datasets/uscensus/schema.sql

```diff
@@ -0,0 +1,71 @@
+DROP SCHEMA IF EXISTS uscensus CASCADE;
+
+CREATE SCHEMA uscensus;
+
+/* Create and define metadata tables */
+CREATE TABLE uscensus.acs_variables (
+    "id" SERIAL UNIQUE,
+    "year" INTEGER,
+    "product" CHAR(4),
+    "census_id" VARCHAR(20),
+    "label" VARCHAR(350),
+    PRIMARY KEY ("year", "product", "census_id")
+);
+COMMENT ON TABLE uscensus.acs_variables IS E'
+Table Description:
+
+This table contains information on the variables that are listed for one of the ACS products (1 and 5 year estimates):
+
+For more information on these variables, please refer to the US Census page about the [ACS 1 year estimates](https://www.census.gov/data/developers/data-sets/acs-1year.html) and the [ACS 5 year estimates](https://www.census.gov/data/developers/data-sets/acs-5year.html)
+
+Words of caution:
+
+Source: US Census';
+
+COMMENT ON COLUMN uscensus.acs_variables.id IS E'An internal identifier for referencing different variables.';
+COMMENT ON COLUMN uscensus.acs_variables.year IS E'The year this product was collected/published';
+COMMENT ON COLUMN uscensus.acs_variables.product IS E'Which ACS product this variables comes from. Can take the values `acs1` or `acs5`';
+COMMENT ON COLUMN uscensus.acs_variables.census_id IS E'The variable name according to the Census documentation.';
+COMMENT ON COLUMN uscensus.acs_variables.label IS E'A short label that describes what the variable includes.';
+
+
+CREATE TABLE uscensus.bfs_variables (
+    "id" INTEGER,
```

Comment:
We should have a primary key for this table

Reply:
Yes. I probably should have just cut table out of this PR... Entire table is gone now.